README.md

```diff
@@ -1,84 +1,130 @@
 Yii2 Captcha Extension
 ======================
 
-[Gregwar's Captcha library](https://github.com/Gregwar/Captcha) wrapper for Yii2.
+[![Latest Stable Version](https://img.shields.io/packagist/v/juliardi/yii2-captcha?label=stable)](https://packagist.org/packages/juliardi/yii2-captcha)
+[![Total Downloads](https://img.shields.io/packagist/dt/juliardi/yii2-captcha)](https://packagist.org/packages/juliardi/yii2-captcha)
+[![Latest Stable Release Date](https://img.shields.io/github/release-date/juliardi/yii2-captcha)](https://github.com/juliardi/yii2-captcha)
+[![License](https://img.shields.io/github/license/juliardi/yii2-captcha)](https://github.com/juliardi/yii2-captcha)
 
-Installation
-------------
+> Yii2 Captcha uses [Gregwar's Captcha library](https://github.com/Gregwar/Captcha) wrapper for Yii2.
 
-The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
-Either run
-```
-php composer.phar require --prefer-dist juliardi/yii2-captcha "*"
-```
-or add
+## Table of Contents
+
+- [Yii2 Toastr](#yii2-toastr)
```

Comment:
Nama package-nya masih belum sesuai, masih yii2-toastr, harusnya yii2-captcha. Table of contents-nya juga masih belum sesuai dengan isi file README ini.